src/pages/pass-finalisation/cat-home-test/pass-finalisation.cat-home-test.page.html

```diff
@@ -30,15 +30,6 @@
           (licenseNotReceived)="provisionalLicenseNotReceived()">
         </license-provided>
 
-        <transmission
-          [formGroup]="form"
-          [transmission]="pageState.transmission$ | async"
-          (transmissionChange)="transmissionChanged($event)">
-        </transmission>
-
-        <warning-banner *ngIf="displayTransmissionBanner()" warningText="An automatic licence will be issued.">
```

Comment:
You will want to remove this mock from the tests. Can't see any more in the file

Reply:
warning banner used for d255 warning so it's fine to remain.